### DISCUSSION TITLE
AUTOMATIC1111's stable-diffusion-webui has been removed....  (???????)

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#2020] [Q&A]
bonjour je viens de tomber sur ça (dans les infos de versions d'un autre notebook  SD pour Google colab ) 

"AUTOMATIC1111's stable-diffusion-webui has been removed because Google Colab has banned the use of strings named stable-diffusion-webui due to massive use of webui. "

https://github.com/Linaqruf/sd-notebook-collection

ça veut dire quoi ??? que qu'automatic1111 est désormais interdit par google colab ???

**Comment by TheLastBen** [comment]
c pas interdit, ils veulent juste pousser les gens à payer
